packages/core-bridge/src/lib.rs

```diff
@@ -900,8 +903,31 @@ fn worker_shutdown(mut cx: FunctionContext) -> JsResult<JsUndefined> {
     }
 }
 
+/// Convert Rust SystemTime into a JS array with 2 numbers (seconds, nanos)
+fn system_time_to_js<'a, C>(cx: &mut C, time: SystemTime) -> NeonResult<Handle<'a, JsArray>>
```

Comment:
Yeah, per my other comment feels like we could maybe just avoid this whole thing

Reply:
If we're willing to compromise on the ordering.